ipsec_offload_plugin/ipsec_offload/ipsec_offload.c

```diff
@@ -909,31 +958,45 @@ METHOD(kernel_ipsec_t, del_policy, status_t,
 			     "add entry failed err_code[ %d]", err);
 
 		if (data->sa->mode == MODE_TUNNEL) {
+#ifdef STANDALONE_IPSEC_RECIPE			
 			err = ipsec_outer_ipv4_decap_mod_table(IPSEC_TABLE_DEL,
```

Comment:
We don't need encap/decap tables ?? How it is handled in tunnel case??

Reply:
For decap, we have a direct decap mod action which doesn't need any parameters instead of decap table. Please check V3 P4 for reference